code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -641,3 +641,6 @@
 /mob/living/simple_animal/Login()
 	..()
 	walk(src, 0) // if mob is moving under ai control, then stop AI movement
+
+/mob/living/simple_animal/proc/npc_safe(mob/user)
+	return FALSE
```

Comment:
It would be nice if this was either named something more descriptive like `can_respawn_as()` for example, or there was a brief comment explaining what it does.